packages/nimble-components/src/anchor/index.ts

```diff
@@ -0,0 +1,51 @@
+import { attr } from '@microsoft/fast-element';
+import {
+    DesignSystem,
+    Anchor as FoundationAnchor,
+    anchorTemplate as template,
+    applyMixins,
+    DelegatesARIALink,
+    AnchorOptions
+} from '@microsoft/fast-foundation';
+import { styles } from './styles';
+import type { AnchorAppearance } from './types';
+
+declare global {
+    interface HTMLElementTagNameMap {
+        'nimble-anchor': Anchor;
+    }
+}
+
+/**
+ * A nimble-styled anchor
+ */
+export class Anchor extends FoundationAnchor {
+    /**
```

Comment:
The spec should be updated to reflect api changes ie disabled was removed from anchor

Reply:
Updated